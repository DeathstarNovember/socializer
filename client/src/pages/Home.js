import React from "react";
import { Col, Container, Row } from "react-bootstrap";
import { Helmet } from "react-helmet";
import { ChatBar, Posts } from "components";

const Home = () => {
  return (
    <>
      <Helmet>
        <title>Socializer</title>
      </Helmet>
      <Container>
        <Row>
          <Col md={4} className="d-none d-md-block">
            <ChatBar />
          </Col>
          <Col xs={12} md={8}>
            <Posts />
          </Col>
        </Row>
      </Container>
    </>
  );
};

export default Home;
