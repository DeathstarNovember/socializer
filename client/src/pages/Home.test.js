import React from "react";
import { render } from "@testing-library/react";
import Home from "./Home";

jest.mock("components", () => ({
  ChatBar: () => <div className="chat-bar" />,
  Posts: () => <div className="posts" />,
}));

describe("Home", () => {
  it("renders correctly", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
