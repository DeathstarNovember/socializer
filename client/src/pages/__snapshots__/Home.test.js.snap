// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="row"
    >
      <div
        class="d-none d-md-block col-md-4"
      >
        <div
          class="chat-bar"
        />
      </div>
      <div
        class="col-md-8 col-12"
      >
        <div
          class="posts"
        />
      </div>
    </div>
  </div>
</div>
`;
